app/controllers/responses_controller.rb

```diff
@@ -8,4 +8,13 @@ def show
     response = Response.find params[:id]
     render json: ResponseSerializer.new(response).as_json
   end
+
+  def update
+    response_finished = FinishResponse.run response_uuid: params[:id]
+    if response_finished.valid?
+      render json: { result: 'ok', message: 'Thanks!' }
```

Comment:
Renderen van json heeft geen betekenis want er wordt niets mee gedaan. Liever:
- `head :ok` als de interaction valid is
- `head :unprocessable_entity` als de interaction invalid is